packages/components/src/Components/TagModal/TagModal.js

```diff
@@ -83,10 +83,19 @@ class TagModal extends Component {
                 variant="medium"
                 {...onApply && {
                     actions: [
-                        <Button key="confirm" variant="primary" onClick={(e) => {
-                            onApply();
-                            toggleModal(e, true);
-                        }}>
+                        <Button
+                            key="confirm"
+                            variant="primary"
+                            isDisabled={
+                                isTabbed ?
+                                    Object.values(selected || {}).every(values => !values || values?.length === 0) :
+                                    !selected || selected?.length === 0
+                            }
+                            onClick={(e) => {
+                                onApply();
+                                toggleModal(e, true);
```

Comment:
This event passing could potentially trigger the react synthetic event console error when used in async way (don't know if that is possible).

You can avoid that calling `event.persist()` before passing it to the `toggleModal` callback.